sources/packages/web/src/components/common/modals/ConfirmModal.vue

```diff
@@ -16,6 +16,7 @@
         @primaryClick="resolvePromise(true)"
         @secondaryClick="resolvePromise(false)"
         :disablePrimaryButton="disablePrimaryButton"
+        :showSecondaryButton="showSecondaryButton"
```

Comment:
what is the scenario, where we will not show the "cancel/back" Secondary Button? for my info

Reply:
It's for this scenario:
![image](https://user-images.githubusercontent.com/78114138/202529521-299bbbe9-1f2c-4577-ba29-38405c4b465a.png)